tests/lib/rules/no-component-lifecycle-hooks.js

```diff
@@ -124,5 +147,61 @@ ruleTester.run('no-component-lifecycle-hooks', rule, {
         },
       ],
     },
+    {
+      code: `
+        import Component from '@ember/component';
+
+        export const Component2 = Component.extend({
+          didDestroyElement() {},
+        });
+
+        export const Component1 = Component.extend({
+          willDestroy() {},
+        });
+      `,
+      output: null,
+      errors: [
+        {
+          message: ERROR_MESSAGE,
+          type: 'Identifier',
+        },
+      ],
+    },
+    {
+      code: `
+        import Component from "@ember/component";
+
+        export const Component1 = Component.extend({
+          test: computed('', function () {}),
+          didDestroyElement() {},
+        });
+      `,
+      output: null,
+      errors: [
+        {
+          message: ERROR_MESSAGE,
+          type: 'Identifier',
+        },
+      ],
+    },
+    {
```

Comment:
This looks like a duplicate invalid test case. The only difference is the type of quotes used in the import statement.